firmware/application/apps/ui_recon.cpp

```diff
@@ -624,6 +624,7 @@ ReconView::ReconView(NavigationView& nav)
     };
 
     button_restart.on_select = [this](Button&) {
+        frequency_file_load(true);
         if (frequency_list.size() > 0) {
             def_step = step_mode.selected_index();  // Use def_step from manual selector
             frequency_file_load(true);
```

Comment:
Is this frequency_file_load() on line 630 now redundant?  Setting "def_step" on the line above might be redundant too, since the first thing that frequency_file_load() does is set "def_step".

Reply:
It's used to restore the frequency list if you altered it using button remove in recon mode.

Reply:
That's a mistake. Solved by taking out the second one.

Reply:
How about the redundant "def_step" setting on line 629?  It seems like frequency_file_load() is already setting that, unless I'm missing something.

Reply:
I opened https://github.com/eried/portapack-mayhem/pull/1282 to remove it. I read your previous comment too quick and did not see.